src/components/pages/Categories/List.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div class="category">
+    <app-category-post
+      class="category-post"
+      category=""
+      :access="access"
+    />
+    <app-category-list
+      :target-array="categoryList"
+      :theads="theads"
+      :access="access"
+    />
+  </div>
+</template>
+
+<script>
+import { CategoryList, CategoryPost } from '@Components/molecules';
+import Mixins from '@Helpers/mixins';
+
+export default {
+  components: {
+    appCategoryList: CategoryList,
+    appCategoryPost: CategoryPost,
+  },
+  mixins: [Mixins],
+  data() {
+    return {
+      theads: ['カテゴリー名', '', '', ''],
```

Comment:
[nits]
ここは「カテゴリー名」だけ渡せば良いので

```suggestion
      theads: ['カテゴリー名'],
```

で大丈夫かなと思います！